### PR TITLE
Request to merge Setup

### PR DESCRIPTION
**Pull request by Kingstalux**
- I started building my school library app. 
- In this initial step, I implemented the classes to represent students and teachers.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    if is_of_age? == true || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  def work
+    true
+  end
```
Comment (mikethreels) [inline person.rb]: - [x] please remove this method, it is not part of the requirements and it seems unused as well.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(age, name, parent_permission: true)
+    @classroom = classroom
+  end
```
Comment (mikethreels) [inline student.rb]: - [x] in the initialize method apart from `classroom` you also need to pass `age`, `name` and `parent_permission` as arguments to make it work.

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name, parent_permission: true)
+    @specialization = specialization
+  end
```
Comment (mikethreels) [inline teacher.rb]: - [x] in the initialize method apart from `classroom` you also need to pass `age`, `name` and `parent_permission` as arguments to make it work.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kingsely, 

Good job so far!
highlights:
good job creating the `can_use_services?` method in combination with the `of_age?` method.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (Kingstalux) [reply person.rb]: Thanks sorry, I was trying something there and I forgot to delete it 😅

**Review by D4nielJ — APPROVED** [review]
## Hello @Kingstalux, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**